tests/cachetest_server.py

```diff
@@ -0,0 +1,93 @@
+            
+import asyncio
+import multiprocessing
+import threading
+import uvicorn
+import signal
+from hypercorn.asyncio import serve
+from hypercorn.config import Config
+import time
+
+from httpx import AsyncClient
+from fastapi import FastAPI
+from devops_sccs.cache import *
```

Comment:
```suggestion
from devops_sccs import cache
```
Personnellement, je prefere cette approche pour ce que ca procure en clarite (namespacing)